node/tessel-export.js

```diff
@@ -427,6 +434,12 @@ Tessel.Pin.interruptModes = {
   low: 5,
 };
 
+Tessel.Pin.pullModes = {
+  pulldown: 0,
+  pullup: 1,
+  none: 2,
```

Comment:
Can these be "const" named to match the C code?

Either: 

```js
Tessel.Pin.pullModes = {
  PULL_DOWN: 0,
  PULL_UP: 1,
  PULL_NONE: 2,
};
```

or 

```js
Tessel.Pin.pullModes = {
  DOWN: 0,
  UP: 1,
  NONE: 2,
};
```

My vote is for the latter.